kyverno-notation-aws.yaml

```diff
@@ -0,0 +1,49 @@
+#nolint:git-checkout-must-use-github-updates,valid-pipeline-git-checkout-tag
+package:
+  name: kyverno-notation-aws
+  version: 1.0.1_git20241210
+  epoch: 0
+  description: Kyverno extension service for Notation and the AWS signer
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: efda86b8e6a97938c3caafcb5c98a5aede2b97e1
+      repository: https://github.com/nirmata/kyverno-notation-aws
+      branch: main
+
+  - uses: go/bump
+    with:
+      deps: github.com/kyverno/kyverno@v1.13.0 github.com/golang-jwt/jwt/v4@v4.5.1 github.com/open-policy-agent/opa@v0.68.0
+      replaces: github.com/docker/docker=github.com/docker/docker@v26.1.5+incompatible
+
+  - uses: go/build
+    with:
+      modroot: .
+      output: kyverno-notation-aws
+      packages: .
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Kyverno extension service for Notation and the AWS signer (compatibility)"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
```

Comment:
```suggestion

```
nit: we can omit this line given we're already creating a nested directory on the next line.

Reply:
removed brother